CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ CHANGELOG
 
 - Add the .NET Core 3.0 runtime to the pulumi/pulumi container. [#3616](https://github.com/pulumi/pulumi/pull/3616).
 
+- Fix bug in determining PRNumber and BuildURL for an Azure Pipelines CI environment.
```

Comment:
It's OK to not have a specific bug number, but we should at least link to _this_ PR (`[#3677](https://github.com/pulumi/pulumi/pull/3677)`). That way someone can go to learn more about what we mean by "PRNumber and BuildURL for an Azure Pipelines CI environment".

Reply:
Ha! I scrolled down the changelog and couldn't determine if we are doing that always now. I figured I'd let you comment on it. So I guess I'll add the PR next to it.

Reply:
Done.